syntax = "proto3";

package band_proto;

import "band/backend/grpc/proto/common.proto";
import "band/backend/grpc/proto/request.proto";
import "band/backend/grpc/proto/response.proto";
import "band/backend/grpc/proto/model.proto";

service BandService {
    rpc RequestSync(Request) returns (Response) {}
    rpc GetModelDesc(Void) returns (ModelDescriptor) {}
    rpc CheckModelDesc(ModelDescriptor) returns (Response) {}
}